membership-attribute-service/test/services/AttributesFromZuoraTest.scala

```diff
@@ -157,7 +157,7 @@ class AttributesFromZuoraTest(implicit ee: ExecutionEnv) extends Specification w
         val attributes: Future[(String, Option[Attributes])] = attributesFromZuora.getAttributes(testId, identityIdToAccountIds, subscriptionFromAccountId,  paymentMethodResponseNoFailures, mockDynamoAttributesService, referenceDate)
         attributes must be_==("Zuora", Some(contributorAttributes)).await
 
-        there was one(mockDynamoAttributesService).update(contributorAttributesWithTtl)
+        eventually(there was one(mockDynamoAttributesService).update(contributorAttributesWithTtl))
```

Comment:
I'm undecided if using eventually matchers is the best thing to do in these tests. Any other ideas?